routes/README.md

```diff
@@ -43,7 +43,7 @@ The route groups are implemented as hapi plugins themselves.
 
 - [session.js](session.js)
 - [account.js](account.js)
-- [profile.js](profile.js) ([tbd hoodie-server-account#11](https://github.com/hoodiehq/hoodie-server-account/issues/11))
+- [profile.js](profile.js)
```

Comment:
ah, very good catch! @patriciagarcia I to totally missed that. Can we move the `GET /session/account/profile` into the new profile.js file?